core/src/main/java/nl/nn/adapterframework/webcontrol/api/BrowseJdbcTable.java

```diff
@@ -138,23 +141,23 @@ public Response execute(LinkedHashMap<String, Object> json) throws ApiException
 			try (Connection conn =qs.getConnection()) {
 				ResultSet rs = null;
 				try {
-					rs = conn.getMetaData().getColumns(null, null, tableName, null);
+					rs = qs.getDbmsSupport().getTableColumns(conn, tableName);
 
 					if (!rs.isBeforeFirst()) {
 						rs.close();
-						rs = conn.getMetaData().getColumns(null, null, tableName.toUpperCase(), null);
+						rs = qs.getDbmsSupport().getTableColumns(conn, tableName.toUpperCase());
 					}
```

Comment:
Zou dit er uit kunnen?
Zie commit: https://github.com/ibissource/iaf/commit/c8c7e62ed656116943b7bbd2f778bab4ba0cb1a0 hoe het ontstaan is. Ik weet niet precies wat bedoeld wordt met 'in case of multiple records'.